sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -864,6 +864,16 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ENABLE_ORC_FLOAT_TYPES_TO_STRING =
```

Comment:
Please add some docs to this that indicate what will happen if we run into this situation.  For most configs when we are in this kind of a situation we fall back to the CPU, but here we will throw an exception and the job will fail.

Reply:
Have updated this in `config.md`.